### PR TITLE
News post for release-20170527.

### PR DESCRIPTION
**Pull request by pchote**
post-fullpage

### CODE REVIEW

File: content/news/release-20170527.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+A collection of smaller fixes are listed in the  [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/).
+
+Head on over to our [download page](/download/) to grab the new release, and spread the word!
+
+</div>
+</div>
+The  [original release-announcement](/news/release-20170421/) for release-20170421 is included below:
+
+<hr>
```
Comment (pchote) [inline content/news/release-20170527.md]: Everything below this line is a direct copy from the original post, so no need to proof read.

File: content/news/release-20170527.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+kind: article
+title: "Release 20170527 hotfix now available"
+author: "Paul Chote"
+created_at: 2017-05-23 20:00 +0000
+disqus_id: "release-20170421"
```
Comment (pchote) [inline content/news/release-20170527.yaml]: Note: this is intentional.  I don't see any need to reset the comments for these minor releases.

File: content/news/release-20170527.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+<div style="margin: -10px 5px" markdown="1">
+We are today pushing a new bugfix release that addresses a collection of issues that were missed during the playtesting phase of release-20170421.
+
+The main changes in this hotfix are:
+
+* Disabled HiDPI support on Windows, fixing the graphical glitches this caused on non-HiDPI displays.
+* Fixes cursor glitches introduced with the new Mono 5.0 runtime.
+* Fixes graphical glitches with the Fullscreen (Legacy) mode on macOS.
+
+A collection of smaller fixes are listed in the  [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/).
```
Comment (pchote) [inline content/news/release-20170527.md]: The changelog link will be updated once we have a release tagged.

File: content/news/release-20170527.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+Head on over to our [download page](/download/) to grab the new release, and spread the word!
+
+</div>
+</div>
+The  [original release-announcement](/news/release-20170421/) for release-20170421 is included below:
```
Comment (abcdefg30) [inline content/news/release-20170527.md]: Double whitespace after `The`.

File: content/news/release-20170527.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+We are today pushing a new bugfix release that addresses a collection of issues that were missed during the playtesting phase of release-20170421.
+
+The main changes in this hotfix are:
+
+* Disabled HiDPI support on Windows, fixing the graphical glitches this caused on non-HiDPI displays.
+* Fixes cursor glitches introduced with the new Mono 5.0 runtime.
```
Comment (abcdefg30) [inline content/news/release-20170527.md]: You're using the past tense above ("disabled"), but present here. I'd suggest changing the "fixes" here and below to "fixed".

**Comment by pchote** [comment]
Updated with correct dates, changelog link, and release ID.

**Pull request merged**